src/neuroconv/schemas/base_metadata_schema.json

```diff
@@ -143,7 +142,7 @@
             "U",
             "O"
           ],
-          "default": "U"
+          "default": ""
```

Comment:
Also including a fix for the default placeholder for `sex`, but I'm not entirely sure if the form will complain about a `default` value that is not in the `enum`

@garrettmflynn

Reply:
At least for the form, this is equivalent to no default value. The final results object will have no "sex" key when an empty string is registered in the input.

Is there a reason you need a default value defined at all?

Reply:
No, removing it is the goal - I just wasn't sure what the forms expect